crypto/ripemd/build.info

```diff
@@ -19,10 +19,10 @@ SOURCE[../../libcrypto]=rmd_dgst.c rmd_one.c $RMD160ASM
 DEFINE[../../libcrypto]=$RMD160DEF
 
 # When all deprecated symbols are removed, libcrypto doesn't export the
-# RIPEMD160 functions, so we must include them directly in liblegacy.a
+# RIPEMD160 functions, so we must include them directly in libcommon.a
 IF[{- $disabled{'deprecated-3.0'} -}]
-  SOURCE[../../providers/liblegacy.a]=rmd_dgst.c rmd_one.c $RMD160ASM
-  DEFINE[../../providers/liblegacy.a]=$RMD160DEF
+  SOURCE[../../providers/libcommon.a]=rmd_dgst.c rmd_one.c $RMD160ASM
+  DEFINE[../../providers/libcommon.a]=$RMD160DEF
```

Comment:
Keep this as it was.

Reply:
Otherwise you would include RIPEMD code in the FIPS provider.

Reply:
The build fails without this change while building libcommon.a

It won't end up in the FIPS provider because we don't use the whole archive option when linking fips.so.